experiments/functional/jiva-volume-resize/README.md

```diff
@@ -0,0 +1,36 @@
+## Experiment Metadata
+
+| Type       | Description                                                  | Storage | Applications | K8s Platform |
+| ---------- | ------------------------------------------------------------ | ------- | ------------ | ------------ |
+| Functional | Increase the storage capacity of jiva volumes. | OpenEBS | Any          | Any          |
+
+## Entry-Criteria
+
+- K8s nodes should be ready.
+- K8s nodes should have ubuntu 18.04 installed.
+- Application should be deployed using 'openebs jiva default' storage class.
```

Comment:
Application should be deployed using jiva storage engine

Reply:
done